src/azure-cli/azure/cli/command_modules/cdn/_params.py

```diff
@@ -135,6 +135,33 @@ def load_arguments(self, _):
     with self.argument_context('cdn custom-domain create') as c:
         c.argument('location', validator=get_default_location_from_resource_group)
 
+    with self.argument_context('cdn custom-domain enable-https') as c:
+        c.argument('profile_name', id_part=None, help='Name of the parent profile.')
```

Comment:
Prefer ['--name', '-n’ ]e

Reply:
The name_arg_type parameter to the argument custom_domain_name already makes the parameter use --name, -n as requested. See the example for `cdn custom-domain enable-https`